pkg/api/logger.go

```diff
@@ -10,7 +12,7 @@ type requestLogger struct{}
 // Print outputs messages with priority "info" by default
 func (r *requestLogger) Print(v ...interface{}) {
 	logger := logging.Logger()
```

Comment:
Why instantiate a new logger each time? Maybe just add a private field to the `requestLogger` instead and reuse that one?

Reply:
I'll shoot this section of the code in the face anyhow soon. Don't worry ;)